### ISSUE TITLE
Documents are often underrepresented in sample, but overrepresented in variety

### ISSUE THREAD

**Issue by hidde** [w3c/wai-wcag-em#25]
In most WCAG-EM-based accessibility conformance reports about websites that we see at work, the sample includes mostly web pages and only 1 or 2 documents (eg PDF).

For sampled web pages, it's common that there a 10s, 100s or 1000s of pages on the site that are similar, for instance because they use the same template. The documents are commonly one-offs, there is a lot more variety between the documents.

This probably means we get a more even-handed look at the accessibility of the web pages, than we do of the documents. 

Should we add recommendations in WCAG-EM to account for this?

**Comment by JeroenHulscher** [comment]
As WCAG-EM is technology agnostic, and this idea should be part of policy within organizations, we think we did what we could in terms of broadening the scope of WCAG-EM to be able to include documents as well. Will close this issue for now, but surely not ignoring the great remarks by @iacobien.
